python/cugraph/cugraph/centrality/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources
+    betweenness_centrality_wrapper.pyx
+    edge_betweenness_centrality_wrapper.pyx
+)
+set(linked_libraries cugraph::cugraph)
+
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX centrality_
+)
+
+set(targets_using_numpy centrality_betweenness_centrality_wrapper centrality_edge_betweenness_centrality_wrapper)
+foreach(target IN LISTS targets_using_numpy)
+  target_include_directories(${target} PRIVATE "${Python_NumPy_INCLUDE_DIRS}")
```

Comment:
What defines `Python_NumPy_INCLUDE_DIRS`?

Reply:
https://cmake.org/cmake/help/latest/module/FindPython.html#result-variables

Reply:
Sorry to be clear I meant where is the related call to find_package(Python)? 
I see one in `python/pylibcugraph/pylibcugraph/CMakeLists.txt` but I presume that doesn't have an impact here.

Maybe the real issue is the one in `python/pylibcugraph/pylibcugraph/CMakeLists.txt` is not needed at all and is confusing me

Reply:
Oh that is a great question, I hadn't even gotten around to that in my first pass of review. @jnke2016 in all your scripts, how did you decide what to add to the `targets_using_numpy` list? From a quick survey, it looks like you literally just put all targets into that list? My guess is that right now nothing in cugraph actually requires NumPy's C-API, so Rob is absolutely right that nothing is actually doing a `find_package(Python)` and then `target_include_directories` is probably just getting an empty string and not adding anything, but it's not actually causing problems because nothing needs NumPy header files. It's very likely that we can remove those altogether.

Reply:
I was getting confused by the python `numpy` and the `numpy C-API` (which cugraph does not need).